### ISSUE TITLE
Missing Data: Minnesota Legislation

### ISSUE THREAD

**Issue by gboone** [opendata/Open-Data-Census#56]
STATE: MN
DATASET: Legislation
URL: see comments

Bills before the legislature, and the legislators that comprise it.

Exists: yes
Digitized: yes
Public: yes
Without cost: yes
Online: yes
Machine-readable: no
Available in bulk: no
No license restrictions: yes
Up-to-date: yes
In the state repository: no
Verifiable: sometimes
Complete: no

Comments:
There are a few different sources but all the information we're looking for is available making this a bit tricky. While all the information is available, I'm marking complete "no" because of this:
* Data about house reps is available here: http://www.house.leg.state.mn.us/members/hmem.asp, an excel spreadsheet is available
  * "Date sworn in" is only available on an individual member's page (example: http://www.house.leg.state.mn.us/members/members.asp?id=15390)
* Information about senators is available here: http://www.senate.mn/members/index.php?ls=#alpha, a download of all the senatorial information is not available
  * Navigating to `www.senate.mn/senator{{lastname}}` will get you to an individual senator's page. In theory you could scrape those pages to get all the information.
* There's also [this PDF](http://www.house.leg.state.mn.us/hinfo/leginfo/elecdir.pdf) with information about membership of both houses as of the most recent election.
* Information about bills _is_ available in a machine readable format for each house as a `|` delimited database file but will be hard to parse since the first line is not column names. [House](https://www.revisor.mn.gov/billstatus/hstat_db), [Senate](https://www.revisor.mn.gov/billstatus/sstat_db). This is the only source available over HTTPS by default.

I'm marking machine readable as a no because while scraping HTML and parsing PDFs are possible, it is vastly more effort than consuming structured data files. Where those files exist, they are incomplete.

**Comment by waldoj** [comment]
[I've got this up on the census now](https://census.usopendata.org/datasets.html?state=Minnesota)—thank you!
